docs/SparkboxPRChecklist.md

```diff
@@ -0,0 +1,77 @@
+# Sparkbox Pull Request Checklist
+
+**When reviewing a pull request, please check for the following:**
+## Code
+#### Code Quality
+- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
+- [ ] Are there any code smells? https://blog.codinghorror.com/code-smells/
+
+#### Tests
+- [ ] Unit test provided
+- [ ] Does this require a behat test?
+
+#### Documentation
+- [ ] Are there enough comments inline with the code?
+- [ ] Is there more documentation needed?
+
+#### Accessibility
+- [ ] Are markup and styles accessible?
+
+#### Security
+- [ ] Do forms need to be sanitized?
+- [ ] Any obvious security flaws or potential holes?
+
+#### Internationalization
+is this happening as of right now? 
+- [ ] Is all content (labels and other strings) translated?
```

Comment:
@sherakama could you clarify how you and SWS are generally applying the `trans` or translate filter to UI text? What cases are important to cover, or what's the reasoning behind the use on different sites?

Reply:
Copying from our slack channel so we have it documented here as well

> We follow best practices and run all strings through the translation APIs in PHP and JS when we should and can. I suspect we may have missed some as we don't have any multilingual sites but it is part of our core practice to support multilang

> I'd say stick to the Drupal best practice conventions and run strings through t & trans but don't go to the extent of creating tests for it.